[CI] IndexRecoveryIT testSourceThrottling failing

**Build Scans:**
- [elasticsearch-intake #33953 / part1](https://gradle-enterprise.elastic.co/s/ddmtdp6u7sqri)
- [elasticsearch-pull-request #116421 / part-1](https://gradle-enterprise.elastic.co/s/fksyweaf3qegs)

**Reproduction Line:**
```
./gradlew ":server:internalClusterTest" --tests "org.elasticsearch.indices.recovery.IndexRecoveryIT.testSourceThrottling" -Dtests.seed=B0CE3572798FC002 -Dtests.locale=lkt-US -Dtests.timezone=Asia/Famagusta -Druntime.java=25
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.indices.recovery.IndexRecoveryIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!('testSourceThrottling'),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: Throttling should be =0 for 'node_t1'. Node stats: {
  "_nodes" : {
    "total" : 1,
    "successful" : 1,
    "failed" : 0
  },
  "cluster_name" : "TEST-TEST_WORKER_VM=[451]-CLUSTER_SEED=[7759066867691907616]-HASH=[11EDAE14BFB]-cluster",
  "nodes" : {
    "x5Kd1YzOQKKiPrO4MsrAfQ" : {
      "timestamp" : 1769714846890,
      "name" : "node_t1",
      "transport_address" : "127.0.0.1:26862",
      "host" : "127.0.0.1",
      "ip" : "127.0.0.1:26862",
      "roles" : [
        "data",
        "data_cold",
        "data_content",
        "data_frozen",
        "data_hot",
        "data_warm",
        "ingest",
        "master",
        "ml",
        "remote_cluster_client",
        "transform"
      ],
      "indices" : {
        "recovery" : {
          "current_as_source" : 0,
          "current_as_target" : 0,
          "throttle_time" : "208.5ms",
          "throttle_time_in_millis" : 208
        }
      }
    }
  }
}
Expected: <0L>
     but: was <208L>
```

**Issue Reasons:**
- [main] 2 failures in test testSourceThrottling (0.2% fail rate in 801 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -88,9 +88,6 @@ tests:
 - class: org.elasticsearch.xpack.esql.qa.single_node.EsqlSpecForceStoredLoadingIT
   method: test {csv-spec:ints.convertStringToBaseIndexGroup7}
   issue: https://github.com/elastic/elasticsearch/issues/141375
-- class: org.elasticsearch.indices.recovery.IndexRecoveryIT
-  method: testSourceThrottling
-  issue: https://github.com/elastic/elasticsearch/issues/141548
 - class: org.elasticsearch.search.aggregations.metrics.TopHitsIT
   method: testMixedSortFieldTypes
   issue: https://github.com/elastic/elasticsearch/issues/142077

--- a/server/src/internalClusterTest/java/org/elasticsearch/indices/recovery/IndexRecoveryIT.java
+++ b/server/src/internalClusterTest/java/org/elasticsearch/indices/recovery/IndexRecoveryIT.java
@@ -740,8 +740,11 @@ public class IndexRecoveryIT extends AbstractIndexRecoveryIntegTestCase {
             .getStore()
             .size();
 
-        logger.info("--> starting node B with default settings");
-        final String nodeB = internalCluster().startNode();
+        logger.info("--> starting node B");
+        final String nodeB = internalCluster().startNode(
+            // Ensure that the target node has a high enough recovery max bytes per second to avoid any throttling
+            Settings.builder().put(RecoverySettings.INDICES_RECOVERY_MAX_BYTES_PER_SEC_SETTING.getKey(), "200mb")
+        );
 
         long chunkSize = Math.max(1, shardSize.getBytes() / 10);
         logger.info(

